.github/workflows/build.yml

```diff
@@ -36,13 +36,11 @@ jobs:
         --batch-mode
         --global-toolchains ${{ github.workspace }}/.github/workflows/toolchains.xml
         --threads 1C
-        -Pbuild-individual-bundles
-        -Pbree-libs
         -Papi-check
         -Dcompare-version-with-baselines.skip=false
         -Dproject.build.sourceEncoding=UTF-8
```

Comment:
```suggestion
```
The `sourceEncoding` is specified in the parent-pom as well and therefore can be removed.